general.rst

```diff
@@ -11,3 +11,4 @@ General
    general/unsplash
    general/base_import
    general/auth
+   general/voip
```

Comment:
For the forward port in ulterior version, this line can be put after **Payment Acquirers** and before **In-App Purchase**